nl/tested-judge/standalone/README.md

```diff
@@ -0,0 +1,170 @@
+wanneer je linting wenst te gebruiken.
+:::
+::: tab Haskell
+De Haskell-oefening vereisen dat de Haskell compiler `ghc` beschikbaar is van uit het `PATH`.
+De installatie van de Haskell compiler kan teruggevonden worden op de [Haskell website](https://www.haskell.org/).
+
+Daarnaast heb je ook de `aeson` bibliotheek nodig, die kan geïnstalleerd worden via het commando:
+```bash
+> cabal v1-install aeson
+```
+
+De Haskell-linter [hlint](https://hackage.haskell.org/package/hlint) moet ook beschikbaar zijn in het `PATH`,
+wanneer je linting wenst te gebruiken.
+:::
+::: tab Java
+De Java-oefeningen vereisen dat Java 11 geïnstalleerd is.
+Zowel de [Oracle versie](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
+als de [OpenJDK versie](https://openjdk.java.net/projects/jdk/11/) kan gebruikt worden.
+
+De commando's `javac` en `java` moeten beschikbaar zijn in het `PATH`.
+
+Java gebruikt de *Checkstyle-8.41* linter, dit is een JAR-bestand die gedownload kan worden op de
+[Checkstyle GitHub pagina](https://github.com/checkstyle/checkstyle/releases/download/checkstyle-8.41/checkstyle-8.41-all.jar).
+De locatie naar de *Checkstyle*-jarbestand moet beschikbaar zijn in de omgevingsvariabele `CHECKSTYLE_JAR`.
+:::
+::: tab JavaScript
+De JavaScript-oefeningen vereisen dat [Node.js v14 of hoger](https://nodejs.org/en/download/) geïnstalleerd is en
+het commando `node` moet beschikbaar zijn in het `PATH`.
+
+Daarnaast wordt er verwacht dat de `abstract-syntax-tree@2.17.6` globaal geïnstalleerd is,
+dit kan met behulp van het commando:
+```bash
+npm install -g abstract-syntax-tree@2.17.6
+```
+
+De JavaScript-linter [eslint](https://eslint.org/) moet ook beschikbaar zijn in het `PATH`,
+wanneer je linting wenst te gebruiken.
+:::
+::: tab Kotlin
+De Kotlin-oefeningen vereisen dat de `kotlin` en `kotlinc` commando's beschikbaar zijn in het `PATH`.
+De installatiemogelijkheden kunnen teruggevonden worden op de
+[Kotlin website](https://kotlinlang.org/docs/command-line.html).
+
+Kotlin gebruikt de *KTLint* linter, dit is een JAR-bestand die gedownload kan worden op de
+[KTLint GitHub pagina](https://github.com/pinterest/ktlint/releases/download/0.41.0/ktlint).
+De locatie naar de *KTLint*-jarbestand moet beschikbaar zijn in de omgevingsvariabele `KTLINT_JAR`.
+:::
+::: tab Python
+Om Python-oefeningen te evalueren moet de Python dependencies geïnstalleerd worden,
+met behulp van het pip commando in de root-directory van TESTed:
+```bash
+> pip3 install -r tested/languages/python/requirements.txt
+```
+
+Python maakt gebruik van [PyLint](https://www.pylint.org/) om de python code te evalueren.
+Deze zal mee geïnstalleerd worden met de Python dependencies. 
+:::
+::::
+
+## 3. Evalueren oefeningen
+TESTed kan uitgevoerd met behulp van het volgende commando in de TESTed root-directory:
+```bash
+> python3 -m tested
+```
+
+Dit commando vereist een JSON-document als standaardinvoer met de volgende velden:
```

Comment:
Perhaps we can link to the config file here? I thought this was the same structure, but unsure

Reply:
This document contains only the `evaluation` field of the config file.